src/material-examples/table-http/table-http-example.ts

```diff
@@ -0,0 +1,70 @@
+import {Component} from '@angular/core';
+import {Http, Response} from '@angular/http';
+import {DataSource} from '@angular/cdk';
+import {Observable} from 'rxjs/Observable';
+import 'rxjs/add/operator/startWith';
+import 'rxjs/add/observable/merge';
+import 'rxjs/add/operator/map';
+
+@Component({
+  selector: 'table-http-example',
+  styleUrls: ['table-http-example.css'],
+  templateUrl: 'table-http-example.html',
+})
+export class TableHttpExample {
+  displayedColumns = ['number', 'state', 'title'];
+  exampleDatabase: ExampleHttpDao | null;
+  dataSource: ExampleDataSource | null;
+
+  constructor(http: Http) {
+    this.exampleDatabase = new ExampleHttpDao(http);
+    this.dataSource = new ExampleDataSource(this.exampleDatabase);
+  }
+}
+
+export interface GithubIssue {
+  number: string;
+  state: string;
+  title: string;
+}
+
+/** An example database that the data source uses to retrieve data for the table. */
+export class ExampleHttpDao {
```

Comment:
I think we should include sorting and pagination in this example. The github api supports it.